src/components/table-static/TableStatic.vue

```diff
@@ -25,13 +25,23 @@
           class="table-static__header"
           data-testid="table-static-header"
           :style="field.width ? { width: withUnit(field.width) } : {}"
-          :class="field.thClass"
-          :data-header="field.key">
+          :class="[field.thClass, { 'table-static__header--sortable': sortable && field.sortable }]"
```

Comment:
@adenvt 
Even if the sortable prop is provided when calling the Table component, but the field is not sortable, it will not be sorted.